packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -138,9 +137,10 @@ export interface GraphQLSitemapServiceConfig
   apiKey: string;
 
   /**
-   * Names of the configured sites
+   * Name of the site to retrieve site paths for
+   * Make it optional
    */
-  sites: string[];
+  siteName?: string;
```

Comment:
This should be back to being required for this service (can just remove `siteName` from the `Omit` above). We can probably drop the check/RangeError as well? Guess it would at least catch empty string.

Reply:
I made this optional, so that multisite service constructor does not require a siteName, uses only the sites array instead. Is there a better way to use single site for parent class (graphql-sitemap-service) and only use site array for child (multisite)?